setup/ProjectManipulator.rb

```diff
@@ -74,6 +74,8 @@ def remove_demo_project
       podfile_path = project_folder + "/Podfile"
       podfile_lines = File.read(podfile_path).lines
       3.times do  podfile_lines.delete_at 2 end
+      podfile_lines.delete_at 4 # inherit-statement
```

Comment:
@Coeur Because there is one line less, we need `podfile_lines.delete_at 3`. Rest looks good.